README.md

```diff
@@ -1,3 +1,50 @@
 # `create-web-ext ui tool`
 
-[`create-web-ext-ui`](http://www.webextensions.tech) is a scaffolding tool for browser extensions. It takes input few parameters and downloads a complete boilerplate browser extension project. This folder will contain all the required folders and files to get started with the extension.
\ No newline at end of file
+Build yourself a browser extension in 15sec. Check it out [here](http://webextensions.tech)
+
+
+[`create-web-ext-ui`](http://www.webextensions.tech) is a scaffolding tool for browser extensions. It is a simple way to jump start your browser extension development. Just select the type of extension you want, and you're off.
+
+Why does this exist?
+--------------------
+Because We got super tired creating the file structure for a new extension over and over again. It doesn't use server side code! Don't believe me? Checkout the github repo.
```

Comment:
checkout github repo :/ you already are on github repo where you are reading this